packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -286,26 +283,31 @@ const CompleteResetPasswordContainer = ({
         );
 
         // TODO add frontend Glean event for successful reset?
-        notifyClientOfSignin(accountResetData);
-
-        sensitiveDataClient.setData('accountResetData', accountResetData);
-
-        // we cannot create a new recovery key if the session is not verified
-        if (accountResetData.verified) {
-          await account.refresh('account');
-          const createRecoveryKeyResult = await account.createRecoveryKey(
-            newPassword
-          );
-          sensitiveDataClient.setData(
-            'newRecoveryKeyData',
-            createRecoveryKeyResult
+        await notifyClientOfSignin(accountResetData);
+
+        // Mobile will automatically close the web view after notifyClientOfSignin
+        // is called. We don't want to automatically generate a recovery key for these
+        // users because they won't be able to see it.
```

Comment:
Ahhh good catch. I think this is going to be a problem potentially, I do see that copy here 😩: https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-auth-server/index.html?path=/story/fxa-emails-templates-passwordresetaccountrecovery--password-reset-account-recovery

but I haven't checked the code,

Reply:
Actually I think this might be alright (after testing locally and thinking through the code paths a bit more). We're not going through the `handleNavigationWithRecoveryKey` flow if mobile and/or account has 2FA, so the correct email gets sent out. Sorry for false alarm.

Reply:
Following up here, this was a problem that's now fixed.